src/Operators/finitedifference.jl

```diff
@@ -969,7 +1044,6 @@ Supported boundary conditions are:
 struct UpwindBiasedProductC2F{BCS} <: AdvectionOperator
     bcs::BCS
 end
-UpwindBiasedProductC2F(; kwargs...) = UpwindBiasedProductC2F(NamedTuple(kwargs))
```

Comment:
Why is this being deleted?

Reply:
We use `UpwindBiasedProductC2F` in TC ([here](https://github.com/CliMA/TurbulenceConvection.jl/blob/10a7f477995da0c73ebb7535c990a85c17f1f680/src/EDMF_functions.jl#L452))

Reply:
Hmm, it shouldn't be...